sympy/printing/codeprinter.py

```diff
@@ -462,6 +462,16 @@ def _print_Function(self, expr):
 
     _print_Expr = _print_Function
 
+    def _print_Derivative(self, expr):
+        obj, *wrt_order_pairs = expr.args
+        meth_name = '_print_Derivative_%s' % obj.func.__name__
+        pmeth = getattr(self, meth_name, None)
+        if pmeth is None:
+            return self._print_not_supported(expr, extra_info=f"\nPrinter {self.__class__.__name__} has no method: {meth_name}")
+        orders = dict(wrt_order_pairs)
+        seq_orders = [orders[arg] for arg in obj.args]
+        return pmeth(obj.args, seq_orders)
```

Comment:
This assumes that `arg` will be in the orders mapping I think. What happens for e.g. `Derivative(zeta(x**2), x)`?

Reply:
That's a good question. Is that a well formed input? Should we throw when the inner arguments are `Expr` instead of `Symbol`?

Currently `.diff()` gives an expression with `Subs`:
```python
>>> zeta(x**2).diff(x)
2*x*Subs(Derivative(zeta(_xi_1), _xi_1), _xi_1, x**2)
```
and now I discover that this isn't handled by `lambdify`, it would perhaps be useful if we could make that work here as well:
```python
>>> f = lambdift(x, zeta(x**2).diff(x), modules=['mpmath'])
>>> f(2)
...
NameError: name 'Subs' is not defined
```

Reply:
`Subs` are now hopefully supported by `lambdify` and `CodePrinter._print_Derivative` will now require that the arguments of its inner function are all `Symbol`.